### PR TITLE
Adds details to README about indexing and withdrawing ADRs

### PR DESCRIPTION
**Pull request by dylanlott**
This PR updates the ADR README with details about the withdrawal process for an ADR as well as adds a link to the ADR Index for tracking all ADRs.

### CODE REVIEW

File: ADRs/README.md [hunk 1]
```diff
@@ -72,8 +72,10 @@ To contribute a new ADR, please follow these steps:
 3. Fill in the necessary information in the new ADR file, following the template structure.
 4. Use the large language model prompts, if desired, to generate or refine the ADR content.
 5. Submit a pull request to merge your ADR branch into the main branch, including a brief description of the problem and solution.
-6. Request a review from the team members, and address any feedback or suggestions provided.
-7. Once approved, merge the pull request to update the repository with the new ADR.
+6. Add a reference to the [ADR Index](https://www.notion.so/pocketnetwork/1a5893d2183b470b93610d2d9ce84d36?v=91b83d5b44d34ba6abf36f9070c2f3a8&pvs=4) for your pull request with appropriate status.
```
Comment (bryanchriswhite) [inline ADRs/README.md:75]: Cool! :raised_hands: 

Just curious what the rationale originally was for this? I can see value in having a view of all the ADRs in one place for multiple reasons / use cases.
 
I find that it makes the ADRs more browsable because it presents **all** ADRs in one place - improved user experience. Using GitHub UI alone automatically bifurcates the ADRs which can bee browsed together into those which have and have not progressed past the "accepted" state:
  - Browsing in GitHub and the repo itself will only present ADRs which have passed the "in review" state (e.g. "accepted", "withdrawn", "superseded").
  - Applying specific filters in the GitHub pull requests page is likely the most reliable way to collect ADRs which are "in review" or earlier in the lifecycle (e.g. "draft") for browsing.

Maintaining that index does introduce a risk of it falling out of sync; however, I think the benefit outweighs the additional overhead at this point and I also think it's likely that we'll be able to largely (or completely) automate it in future process iterations.

Lastly, my interpretation of the share settings is that this is not currently public to the community; was that intentional?:
image

Did you intend to toggle the "share to web" toggle?

File: ADRs/README.md [hunk 2]
```diff
@@ -72,8 +72,10 @@ To contribute a new ADR, please follow these steps:
...
+7. Request a review from the team members, and address any feedback or suggestions provided.
+8. Once approved, merge the pull request to update the repository with the new ADR.
+9. If rejected or withdrawn, update the document's status as withdrawn, close the pull request, and update the status of the ADR in the [ADR Index](https://www.notion.so/pocketnetwork/1a5893d2183b470b93610d2d9ce84d36?v=91b83d5b44d34ba6abf36f9070c2f3a8&pvs=4).
```
Comment (bryanchriswhite) [inline ADRs/README.md:78]: Do you think we need a "rejected" status? My thinking was that we might get away with just having "withdrawn" but I'm realizing that in the case where we have an unresponsive or irrational author, we would either have to update the status to withdrawn as the reviewer (which feels a little weird) or introduce a "rejected" status.

I intentionally avoided including a "rejected" status because of the negative connotation. After a bit of a back and forth with chatGPT, I had a list of candidate words and then looked at the definitions to find a best fit. "Unsuitable" rose to the top and with the exception of "discarded", I disqualified the rest:

1. **Unsuitable**: Not appropriate, fitting, or suitable for a particular purpose or situation; not matching the requirements or standards.
2. Discarded: To get rid of or throw away something that is no longer wanted, needed, or considered valuable.
1. ~~Neglected~~: To **pay little or no attention to, disregard, or fail to care** for someone or something, often resulting in their being **overlooked or not given proper consideration**.
3. ~~Forsaken~~: To abandon or desert someone or something, leaving them **alone, neglected, or without support or care**.
6. ~~Excluded~~: To **deliberately leave out** or **deny someone or something access, participation, or involvement** in a particular group, activity, or situation.
7. ~~Ignored~~: To deliberately **pay no attention to, disregard, or overlook** someone or something, often by choice or as a result of **indifference**.
8. ~~Abandoned~~: To completely **give up, desert, or forsake** someone or something, **leaving them without any support, care, or protection**.
9. ~~Dismissed~~: To reject, disregard, or wave aside someone or something **as unimportant, insignificant, or not worthy of consideration**.
10. ~~Disregarded~~: To intentionally **ignore, pay no attention to, or overlook** someone or something, often as a result of **neglect or indifference**.

TLDR; I think we should add an "unsuitable" state to handle the case of unresponsive and/or irrational contributors such that we can prevent ADRs from getting stuck in "in review" without requiring a second party (i.e. pokt) to "withdraw" them on behalf of said contributors (not the best look).

File: ADRs/README.md [hunk 3]
```diff
@@ -72,8 +72,10 @@ To contribute a new ADR, please follow these steps:
 3. Fill in the necessary information in the new ADR file, following the template structure.
 4. Use the large language model prompts, if desired, to generate or refine the ADR content.
 5. Submit a pull request to merge your ADR branch into the main branch, including a brief description of the problem and solution.
```
Comment (bryanchriswhite) [inline ADRs/README.md:74]: I think we should clarify a bit around who is expected to do what. I'm noticing that these steps should probably addressed to different roles (author, reviewer/maintainer); something like:

5. Author: Open a PR as a draft ...
6. Author: Add a reference to the ADR Index ...
7. Author: Once it's ready for review (ideally this is clearly defined in some guidelines or docs), update the status to "in review" and mark the pull request as "ready for review"
8. Reviewer: reviews ... (ideally we link to some guidelines or docs)
9. Author: addresses any feedback..
10. Reviewer: Updates the status to "accepted" and merges.
11. (optional) Author: If "withdrawn" update the status respectively and close the PR.
12. (optional) Reviewer: If it becomes clear (e.g. over the course of multiple rounds of review) that an ADR is "unsuitable", update the status respectively and close the PR

I could imagine that it may more sense in the long run for there to be one set of instructions for the author and another for the reviewer; however, in the spirit of iteration and to try to keep things simple for now, I think it's fine to keep them combined.

File: ADRs/README.md (hunk 2 above)
Comment (dylanlott) [reply ADRs/README.md:78]: I think unsuitable is a good replacement. I'll update the wording to match 👍

File: ADRs/README.md (hunk 1 above)
Comment (dylanlott) [reply ADRs/README.md:75]: The original rationale was to have all ADRs in a single place for visibility even after closure of the pull request. There is danger in the two falling out of sync but the alternative is to keep them withdrawn ADRs in a folder.  I'm open to that, but this felt lightweight for now.

I absolutely did mean to share to web, I will do that now 💯

File: ADRs/README.md (hunk 3 above)
Comment (bryanchriswhite) [reply ADRs/README.md:75]: bump?

Comment (Olshansk) [reply ADRs/README.md:75]: +1 to everything here:
1. Feels like a "good enough" iteration as we figure it out
2. I like the clarity
3. NIT: Bold the responsible stakeholder

**Review by Olshansk — CHANGES_REQUESTED** [review]

**Review by bryanchriswhite — APPROVED** [review]
Thanks @dylanlott :raised_hands:

**Review by Olshansk — APPROVED** [review]
Looks great!
